src/main/java/Duke.java

```diff
@@ -1,10 +1,94 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    public static void main(String arguments[]) {
+        int counter = 0;
```

Comment:
Can try renaming the variable to something that better helps understand what the counter is for.